x-pack/plugins/security_solution/cypress/upgrade_integration/import_case.spec.ts

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  ALL_CASES_CLOSED_CASES_STATS,
+  ALL_CASES_COMMENTS_COUNT,
+  ALL_CASES_IN_PROGRESS_CASES_STATS,
+  ALL_CASES_NAME,
+  ALL_CASES_NOT_PUSHED,
+  ALL_CASES_NUMBER_OF_ALERTS,
+  ALL_CASES_OPEN_CASES_STATS,
+  ALL_CASES_REPORTER,
+  ALL_CASES_IN_PROGRESS_STATUS,
+} from '../screens/all_cases';
+import {
+  CASES_TAGS,
+  CASE_CONNECTOR,
+  CASE_DETAILS_PAGE_TITLE,
+  CASE_DETAILS_USERNAMES,
+  CASE_EVENT_UPDATE,
+  CASE_IN_PROGRESS_STATUS,
+  CASE_SWITCH,
+  CASE_USER_ACTION,
+} from '../screens/case_details';
+import { CASES_PAGE } from '../screens/kibana_navigation';
+
+import { goToCaseDetails } from '../tasks/all_cases';
+import { deleteCase } from '../tasks/case_details';
+import { navigateFromKibanaCollapsibleTo, openKibanaNavigation } from '../tasks/kibana_navigation';
+import { loginAndWaitForPageWithoutDateRange } from '../tasks/login';
+import { importCase } from '../tasks/saved_objects';
+
+import { KIBANA_SAVED_OBJECTS } from '../urls/navigation';
+
+const CASE_NDJSON = '7_16_case.ndjson';
+const importedCase = {
+  title: '7.16 case to export',
+  user: 'glo',
+  reporter: 'glo@test.co',
+  tags: 'export case',
+  numberOfAlerts: '2',
+  numberOfComments: '4',
+  description:
+    "This is the description of the 7.16 case that I'm going to import in future versions.",
+  timeline: 'This is just a timeline',
+  status: 'In progress',
+  ruleName: 'This is a test',
+  participants: ['test', 'elastic'],
+  connector: 'Jira test',
+};
+const updateStatusRegex = new RegExp(
```

Comment:
Is it possible to use the `have.text` assertion instead of the `match` with a regex? I feel like the regex is less readable.

Reply:
In this case it is not possible to use the `have.text` since the `ago time` expression could vary depending when the test is executed